packager/hls/public/hls_params.h

```diff
@@ -51,6 +51,9 @@ struct HlsParams {
   /// Same as above, but this overrides the default language for text tracks,
   /// i.e. subtitles or close-captions.
   std::string default_text_language;
+  //  Same as the equivalent field in ChunkingParams, forces segments to 
+  //  begin with stream access points if set to true.
```

Comment:
Update the comment, e.g.

// Indicates that all media samples in the media segments can be decoded without information from other segments.

Reply:
Updated the comment, and updated the golden files.